src/common-functions.vue

```diff
@@ -184,4 +184,51 @@ export function getOverallRankEmoji(
 
   return null;
 }
+
+/**
+ * A constant for Chicago avg. utility costs in the latest data year, which we use to estimate an
+ * upper bound of how much a building would pay for gas and electric
+ */
+export const UtilityCosts = {
+  year: 2021,
+  source: 'https://www.bls.gov/regions/midwest/news-release/AverageEnergyPrices_Chicago.htm',
+  electricCostPerKWh: 0.147,
+  gasCostPerTherm: 1.101,
```

Comment:
This should be `1.192`

![Screenshot from 2023-12-05 20-43-32](https://github.com/vkoves/electrify-chicago/assets/3187531/593ca767-b66b-4409-b0af-c008b86791c0)